x/ibc/core/exported/client.go

```diff
@@ -178,6 +178,8 @@ type ConsensusState interface {
 
 // Misbehaviour defines counterparty misbehaviour for a specific consensus type
 type Misbehaviour interface {
+	proto.Message
```

Comment:
cc @robert-zaremba do you know if this is a state machine breaking change? We need to treat headers and misbehaviour as `Any`s which is why I believe we need to declare this in the interface